core_lib/web_helpers/decorators.py

```diff
@@ -10,31 +10,34 @@
 logger = logging.getLogger(__name__)
 
 
-def handle_exception(func, *args, **kwargs):
+def handle_exception(func, log_exception=True,*args, **kwargs):
     try:
         return func(*args, **kwargs)
     except StatusCodeException as n:
         logger.error(f'handle_exception got StatusCodeException error for function `{func}`')
-        logger.exception(n, exc_info=True)
+        logger.exception(n, exc_info=log_exception)
         return response_message(status=n.status_code)
     except AssertionError as n:
         logger.error(f'handle_exception got AssertionError error for function `{func}`')
-        logger.exception(n, exc_info=True)
+        logger.exception(n, exc_info=log_exception)
         return response_message(status=HTTPStatus.INTERNAL_SERVER_ERROR)
     except ExpiredSignatureError as ese:
         logger.error(f'handle_exception got ExpiredSignatureError error for function `{func}`')
-        logger.exception(ese, exc_info=True)
+        logger.exception(ese, exc_info=log_exception)
         return response_message(status=HTTPStatus.UNAUTHORIZED)
     except BaseException as bx:
         logger.error(f'handle_exception got BaseException error for function `{func}`')
-        logger.exception(bx, exc_info=True)
+        logger.exception(bx, exc_info=log_exception)
         return response_message(status=HTTPStatus.INTERNAL_SERVER_ERROR)
 
 
 class HandleException(object):
-    def __call__(self, func, *args, **kwargs):
+    def __init__(self, log_exception: bool = True):
+        self._log_exception = log_exception
+
+    def __call__(self,func, *args, **kwargs):
         @wraps(func)
         def wrapper(*args, **kwargs):
-            return handle_exception(func, *args, **kwargs)
+            return handle_exception(func,self._log_exception, *args, **kwargs)

```

Comment:
`return handle_exception(func, self._log_exception, *args, **kwargs)`

same here as well a space before the log expectation parameter